e2e-tests/tfm/cypress/fixtures/mock-gef-facilities.js

```diff
@@ -1,6 +1,24 @@
 const CONSTANTS = require('./constants');
+const dateConstants = require('../../../e2e-fixtures/dateConstants');
+
+const facilityEndDateEnabled = Number(Cypress.env('GEF_DEAL_VERSION')) >= 1;
```

Comment:
I _think_ it makes sense for this to be based on `GEF_DEAL_VERSION` but just want to sanity check that this isnt meant to be the `FF_TFM....`